.github/workflows/test-dapp-treasury.yml

```diff
@@ -0,0 +1,82 @@
+name: Test Dapp Treasury
+
```

Comment:
This seems to be identical with `test-dapp-oracle.yml`. Is there a way to configure treasury to use the same yml file?  I thought we did something similar with other packages.

Reply:
Nope, each dapp has its own file. Using the same file for all of them would take some time to figure out, and hasn't risen to the level that we need to spend time optimizing that. I agree that would be better, though.